extensions/openshift-client/runtime/src/main/java/io/quarkus/openshift/client/runtime/OpenShiftClientProducer.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.openshift.client.runtime;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.openshift.client.DefaultOpenShiftClient;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.arc.DefaultBean;
+
+@ApplicationScoped
```

Comment:
Let's make it `@Singleton` maybe?

Reply:
by the way, documentations provides un example with `ApplicationScoped`
should it be changed as well?

Reply:
Yeah, it should be `@Singleton`

Reply:
corrected in doc "https://quarkus.io/guides/writing-extensions"
see https://github.com/quarkusio/quarkus/pull/11982/commits/4e5a98a611343bebeaa19628c067f06fb3b3bb05